app/assets/stylesheets/views/article.scss

```diff
@@ -452,6 +452,8 @@
 
 .multiple_reactions_engagement {
   padding: 0.5em 0;
+  height: 43px;
```

Comment:
This is a small fix to a 1px jump in the title that still persists in this area when loaded async, just picking it up as I'm going.